_config/config.yml

```diff
@@ -1,10 +1,17 @@
 ---
-Name: dashboard
-After: 'framework/*','cms/*'
+Name: dashboardconfig
+After:
+    - '#cmsdefaultadmin'
 ---
-AdminRootController:
-  default_panel: 'DashboardAdmin'
+SilverStripe\Admin\AdminRootController:
+  default_panel: Plastyk\Dashboard\Admin\DashboardAdmin
 
-DashboardAdmin:
+Plastyk\Dashboard\Admin\DashboardAdmin:
   extensions:
-    - DashboardSearchExtension
+    - Plastyk\Dashboard\Extensions\DashboardSearchExtension
+
+SilverStripe\Core\Injector\Injector:
+  Psr\SimpleCache\CacheInterface.myCache:
+    factory: SilverStripe\Core\Cache\CacheFactory
+    constructor:
+      namespace: "myCache"
```

Comment:
@Turnerj I don't know if this is the correct way to call the cache. This is from the documentation. I guess we should change _myCache_ to something more meaningful like _plastykDashboardCache_. What do you think?

Reply:
Yeah, I agree. I saw your other comments where there are currently references to _myCache and I'm thinking like you are that we should change it to something more meaningful.

I think _plastykDashboardCache_ would work. I was wondering whether having "cache" in the namespace name was a little redundant but from a quick look through the documentation, it seems to be what they preference.